mesh_startup.scd

```diff
@@ -1,12 +1,19 @@
-// [OPTIONAL]
-// load remote server options for the mesher
-"mesher_server_options.scd".loadRelative;
+// LAN session with shared server
+// Run on each local client:
+~serverIP = "192.168.0.6";
+~serverPort = 57110;
```

Comment:
I like that this is now configurable. This IP address looks like a magic number to me, and I think would be a dynamic thing depending on the network being used. Can you either add a comment to this explaining where this IP comes from, or something that makes it clear that this is a default value that could change?